src/hotspot/share/gc/shared/stringdedup/stringDedupTable.cpp

```diff
@@ -278,7 +278,7 @@ typeArrayOop StringDedupTable::lookup(typeArrayOop value, bool latin1, unsigned
     if (entry->hash() == hash && entry->latin1() == latin1) {
       oop* obj_addr = (oop*)entry->obj_addr();
       oop obj = NativeAccess<ON_PHANTOM_OOP_REF | AS_NO_KEEPALIVE>::oop_load(obj_addr);
-      if (java_lang_String::value_equals(value, static_cast<typeArrayOop>(obj))) {
+      if (!CompressedOops::is_null(obj) && java_lang_String::value_equals(value, static_cast<typeArrayOop>(obj))) {
```

Comment:
This cannot be a narrowOop, therefore there's no point in using CompressedOops::is_null(), you can use != NULL instead.
Also, I'd change the commit msg to reflect what you did/what the solution is like "Add NULL-check to..." and not what the problem was.